packages/target-electron/src/windows/video-call.ts

```diff
@@ -312,9 +312,7 @@ function openVideoCallWindow<T extends CallDirection>(
     return port1
   })()
 
-  // @ts-expect-error ts(2339) This API will be introduced later.
   if (jsonrpcRemote.rpc.iceServers != undefined) {
-    // @ts-expect-error ts(2339) This API will be introduced later.
```

Comment:
This doesn't do what I did in my merge request, but I guess I can make a follow-up later.

Reply:
We can also merge the video-call from your branch. This was just to avoid the error: src/windows/video-call.ts(315,3): error TS2578: Unused '@ts-expect-error' directive.